foxe/main.c

```diff
@@ -217,13 +270,11 @@ bool listen_input(FILE* f) {
 }
 
 int main(int argc, char* argv[]) {
-    if (argc < 2) {
-        printf("Not enough arguments. foxe takes 1 argument but 0 were given!\nUsage: foxe <file-name>\n");
-        return 1;
-    } else if (argc > 2) {
-        printf("To many arguments. foxe takes 1 argument but %d were given!\nUsage: foxe <file-name>\n", argc - 1);
+    if (argc == 1) {
```

Comment:
The code you deleted here was fine. Maybe make an if else? Then there can only be one or two args.

Reply:
You should probably also check what arg 1 says if there is only one.

Reply:
> The code you deleted here was fine. Maybe make an if else? Then there can only be one or two args.

I will do that later.